src/main/java/org/jabref/gui/entryeditor/CommentsTab.java

```diff
@@ -64,7 +64,7 @@ public CommentsTab(PreferencesService preferences,
                 journalAbbreviationRepository,
                 indexingTaskManager
         );
-        this.defaultOwner = preferences.getOwnerPreferences().getDefaultOwner();
+        this.defaultOwner = preferences.getOwnerPreferences().getDefaultOwner().toLowerCase().replaceAll("[^a-z0-9]", "-");
```

Comment:
When using toLowercase etc you should always pass a Locale, usually Locale.Root 
https://mattryall.net/blog/the-infamous-turkish-locale-bug

Reply:
Thanks, I wasn't aware of that. I've updated the code.